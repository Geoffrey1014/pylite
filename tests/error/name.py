type = 123
print typo
