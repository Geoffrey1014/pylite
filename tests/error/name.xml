lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_const c0
    store_global c1
    load_global c2
    print
    load_const c3
    return
  const_vector
    123.000000
    type
    typo
    None
  guard_vector
  line_number_map 1 1 2 2 1 1
