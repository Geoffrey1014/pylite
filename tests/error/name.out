NameError:
  At name.py, line 2, in __init__()
